docs/bos/queryapi/how-works.md

```diff
@@ -34,6 +34,56 @@ An in-depth, detailed overview of the QueryApi components:
 - **Database:** a Postgres database where the developer's indexer data is stored, using a logical DB per user, and a logical schema per indexer function.
 - **API:** a Hasura server running on Google Cloud Platform exposes a GraphQL endpoint so users can access their data from anywhere.
 
+
+## Provisioning
+
+In summary, QueryAPI provisioning consists of the following steps:
+
+1. Create `database` in [Postgres](#postgres), named after account, if necessary
+2. Add `database` to Hasura
+3. Create `schema` in `database`
+4. Run DDL in `schema`
+5. Track all tables in `schema` in [Hasura](#hasura)
+6. Track all foreign key relationships in `schema` in [Hasura](#hasura)
+7. Add all permissions to all tables in `schema`
+
+:::note
+This is the workflow for the initial provisioning. Nothing happens for the remaining provisions, as QueryAPI checks if it has been provisioned, and then skips these steps.
+:::
+
+:::info
+To check if an Indexer has been provisioned, QueryAPI checks if both the `database` and `schema` exist. This check is what prevents the app from attempting to provision an already provisioned indexer.
+:::
+
+### Historical backfill
+
+When an indexer is created, two processes are triggered:
+
+- real-time - starts from the block the indexer was registered (`block_X`), and will execute the indexer function on every matching block from there on.
+- historical - starts from the configured `start_from_block` height, and will execute the indexer function for all matching blocks up until `block_X`.
+
+The historical backfill process can be broken down in to two parts: _indexed_, and _unindexed_ blocks.
+
+Indexed blocks come from the `near-delta-lake` bucket in S3. This bucket is populated via a DataBricks job which streams blocks from NEAR Lake, and for every account, stores the block heights that contain transactions made against them. This processed data allows QueryAPI to quickly fetch a list of block heights that match the contract ID defined on the Indexer, rather than filtering through all blocks.
```

Comment:
```suggestion
**Indexed** blocks come from the `near-delta-lake` bucket in S3. This bucket is populated via a DataBricks job which streams blocks from NEAR Lake, and for every account, stores the block heights that contain transactions made against them. This processed data allows QueryAPI to quickly fetch a list of block heights that match the contract ID defined on the Indexer, rather than filtering through all blocks.
```
Maybe bold this like you do for unindexed below?